docs/release-notes/8.0.asciidoc

```diff
@@ -8,3 +8,35 @@
 [discrete]
 [[known-issues-8.0.0-rc2]]
 ==== Known issues
+
+*The Data View option might not display in upgraded environments with legacy alerts:*
+
+The *Data View* option might not display in environments with legacy alerts. To resolve this, a user with elevated role privileges must visit the {es-sec-app}, open a page that displays alert data (such as the Overview page), then refresh the page ({pull}121390[#121390]).
```

Comment:
Should we repeat this if it's mentioned in the RN section title?

Reply:
Good point, I don't think repeating it brings any extra value. How about removing it and revising the summary to:
To make the **Data View** option appear, a user with elevated role privileges must visit the {es-sec-app}, open a page that displays alert data (such as the Overview page), then refresh the page.